solutions/src/main/java/ru/sberbank/school/task02/Solution.java

```diff
@@ -14,15 +16,31 @@ public static void main(String[] args) {
         // получаем наш валютный калькулятор
         FxConversionService calculator =
                 calculatorFactory.getFxConversionService(new ExternalQuotesServiceDemo());
+        ExtendedFxConversionService extendedCalculator =
+                calculatorFactory.getExtendedFxConversionService(new ExternalQuotesServiceDemo());
+
+        Beneficiary beneficiary = System.getenv("Beneficiary") == "CLIENT"
+                ? Beneficiary.CLIENT : Beneficiary.BANK;
 
         try {
-            BigDecimal price = calculator.convert(ClientOperation.BUY,
-                    Symbol.USD_RUB, BigDecimal.valueOf(1000));
-            System.out.println(price);
-        } catch (FxConversionException ex) {
-            System.out.println(ex.getMessage());
+            BigDecimal priceUsd = calculator.convert(ClientOperation.BUY,
+                    Symbol.USD_RUB, new BigDecimal(100));
+            System.out.println("USD: " + priceUsd);
+
+            Optional<BigDecimal> priceRub = extendedCalculator.convertReversed(ClientOperation.BUY, Symbol.USD_RUB,
+                    new BigDecimal(100), beneficiary);
+            if (priceRub.isPresent()) {
+                System.out.println("RUB " + priceRub.get());
+            }
```

Comment:
&#39;}&#39; at column 13 should be on the same line as the next part of a multi-block statement (one that directly contains multiple blocks: if/else-if/else, do/while or try/catch/finally).

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1414075142>*